### ISSUE TITLE
TypeError in call to render() with version 0.10.4

### ISSUE THREAD

**Issue by monty700** [elapouya/python-docx-template#320]
I had installed docxtpl v0.10.0 on my Windows 10 computer and it worked with no problems.
A couple of days later I installed docxtpl v0.10.4 on Ubuntu v18.04 and I got the following error in the call to render():
```
Traceback (most recent call last):
  File "test_docxtpl.py", line 10, in <module>
    document_template.render(context)
  File "/usr/local/lib/python2.7/dist-packages/docxtpl/__init__.py", line 309, in render
    xml_src = self.build_xml(context, jinja_env)
  File "/usr/local/lib/python2.7/dist-packages/docxtpl/__init__.py", line 268, in build_xml
    xml = self.render_xml(xml, context, jinja_env)
  File "/usr/local/lib/python2.7/dist-packages/docxtpl/__init__.py", line 238, in render_xml
    dst_xml = self.resolve_listing(dst_xml)
  File "/usr/local/lib/python2.7/dist-packages/docxtpl/__init__.py", line 261, in resolve_listing
    xml = re.sub(r'<w:p(?: [^>]*)?>.*?</w:p>', resolve_paragraph, xml)
  File "/usr/lib/python2.7/re.py", line 155, in sub
    return _compile(pattern, flags).sub(repl, string, count)
  File "/usr/local/lib/python2.7/dist-packages/docxtpl/__init__.py", line 254, in resolve_paragraph
    paragraph_properties = re.search(r'<w:pPr>.*</w:pPr>', m[0])
TypeError: '_sre.SRE_Match' object has no attribute '__getitem__'
```

Here's the code:
```
import docxtpl
import docx.shared

document_template = docxtpl.DocxTemplate("TemplateTest.docx")

context = {}
context['Client1FullName_AllCaps'] = "John Williams Smith".upper()
context['Client2FullName_AllCaps'] = "Mary Sutton Smith".upper()

document_template.render(context)
document_template.save("MergedDocument.docx")
```

I uninstalled version 0.10.4 and installed 0.10.0 and the problem was resolved.

**Comment by elapouya** [comment]
could you provide the docx template please

**Comment by monty700** [comment]
[TemplateTest.docx](https://github.com/elapouya/python-docx-template/files/5436002/TemplateTest.docx)

I attached the Word template. I forgot to mention that I'm using Python 2.7.

**Comment by elapouya** [comment]
Tested with docxtpl v0.11.0 : No problem with python 3.6, but crashes with python 2.7

**Comment by elapouya** [comment]
fixed in v0.11.1
